qa/L0_backend_config/test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Copyright 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+SERVER=/opt/tritonserver/bin/tritonserver
+SERVER_TIMEOUT=20
+source ../common/util.sh
+
+SERVER_ARGS="--model-repository=`pwd`/models --default-max-batch-size=-1"
```

Comment:
`--backend-config=default-max-batch-size=-1`

Reply:
I believe I may have misunderstood the conversation. Do we want to have the command line option "`-default-max-batch-size=<positive_int>`" or do we only want to it to be set with the `--backend-config` flag?

Currently this PR supports on the command line:
- `--default-max-batch-size=<positive_int>`
- `--backend-config=<backend>,default-max-batch-size=<positive_int>`

See the latest commit to the QA tests for clarity.

Reply:
"default-max-batch-size" is used only by the backends. 
See this here: https://github.com/triton-inference-server/tensorflow_backend/pull/50#issuecomment-1075700159

In short support the following:

  1.  --backend-config=default-max-batch-size=<positive_int>
  2.  --backend-config=<backend>,default-max-batch-size=<positive_int>

Reply:
Do we want to support this syntax generally or do we only want to support this syntax for specific global backend config settings? For instance, is --backend-config=,max-batch-size=3 now valid (currently it's not)?

Reply:
Do we want to support this syntax generally or do we only want to support this syntax for specific global backend config settings? For instance, is --backend-config=,max-batch-size=3 now valid (currently it's not)?

Reply:
Talked this out offline. I will be implementing the following:
1.    ` --backend-config=default-max-batch-size=<positive_int>`
2.    `--backend-config=<backend>,default-max-batch-size=<positive_int>`

If two of these options are set, one global and one backend specific, then the backend specific option will take precedence over the global setting.

Reply:
As discussed in the call, these options are specified as a part of --backend-config because they don't impact the triton server core, instead the triton server core just pass it to the backend. And the backend is the entity that consumes this information.